DSharpPlus.Test/TestBot.cs

```diff
@@ -172,6 +172,9 @@ private Task Discord_StickersUpdated(DiscordClient sender, GuildStickersUpdateEv
 
         public async Task RunAsync()
         {
+            await this.Discord.ConnectAsync();
+            return;
```

Comment:
Is there any reason this is still in here?

Reply:
Limitations on testing; can be removed.